src/consensus/blockchain_interface.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Optional
+
+from src.consensus.sub_block_record import SubBlockRecord
+from src.types.sized_bytes import bytes32
+from src.types.sub_epoch_summary import SubEpochSummary
+from src.util.ints import uint32
+
+
+class BlockchainInterface:
```

Comment:
Why is this empty?

Reply:
its an interface, what do you mean empty ? all the classes that work with wallet_blockchain.py / blockchain.py
use this interface, this allows us to move all duplicated code there and use blockcache.py in the tests